support-frontend/assets/pages/paper-subscription-landing/components/content/subsCardTab.jsx

```diff
@@ -9,22 +9,10 @@ import Text from 'components/text/text';
 import GridImage from 'components/gridImage/gridImage';
 import { paperHasDeliveryEnabled } from 'helpers/subscriptions';
 import { Accordion, AccordionRow } from '@guardian/src-accordion';
-import { css } from '@emotion/core';
-import { neutral } from '@guardian/src-foundations/palette';
-import { textSans } from '@guardian/src-foundations/typography';
-import { space } from '@guardian/src-foundations';
+import { accordionContainer } from './tabsStyles';
```

Comment:
Same here as previous point. I think it's best if these two components don't share styles.